test/parallel/test-regress-GH-9819.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const execFile = require('child_process').execFile;
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
+const setup = 'const enc = { toString: () => { throw new Error("xyz"); } };';
+
+const scripts = [
+  'crypto.createHash("sha256").digest(enc)',
+  'crypto.createHmac("sha256", "msg").digest(enc)'
+];
```

Comment:
Mhhh I think I will keep it this way unless someone expresses a strong opposing opinion, but thank you for your feedback :)